backend/src/main/kotlin/app/ehrenamtskarte/backend/freinet/util/FreinetDataTransferPermission.kt

```diff
@@ -0,0 +1,16 @@
+package app.ehrenamtskarte.backend.freinet.util
+
+import app.ehrenamtskarte.backend.common.webservice.FREINET_DEMO_REGION_NAME
+import app.ehrenamtskarte.backend.config.Environment
+import app.ehrenamtskarte.backend.exception.service.ForbiddenException
+
+/**
+ * Validates if Freinet data transfer is permitted for the given environment and region.
+ * - Production environment: allowed for all regions.
+ * - Non-production environments: only allowed for demo region, because freinet has no staging environment.
+ */
+fun validateFreinetDataTransferPermission(environment: Environment, regionName: String) {
+    if (environment != Environment.PRODUCTION && regionName != FREINET_DEMO_REGION_NAME) {
+        throw ForbiddenException()
```

Comment:
❓ Not sure if we should create or throw a separate exception, but since this will not happen on production, i think it should be good enough